examples/amp-web-push/publisher-files/helper-frame.html

```diff
@@ -0,0 +1,60 @@
+<!doctype html>
+<html>
+<!-- AMP Web Push Helper IFrame -->
+
+<head>
+  <meta charset="utf-8">
+  <!--
+    Minified script bundle of helper-frame.js, window-messenger.js, and
```

Comment:
Yes, basically this file can be copy pasted as is. 

Except now its no longer in `examples`, but it's back in the `extensions/amp-web-push/publisher-files` directory (had some trouble with the `gulp` task and it was easier to put it here).

I used to minify it using UglifyJS2's online demo, but that won't scale so we now have a gulp task that runs with `gulp dist` to produce the minified HTML publisher files.